src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/util/MiscUtils.java

```diff
@@ -87,6 +87,19 @@ private static String wrapLine(final String line, final int lineLength) {
         return allLines.toString();
     }
 
+    public static String upperCase(String s) {
+        return upperCase(s, 0);
+    }
+
+    /**
+     * Makes all characters in the string uppercase, other than the given index.
+     */
+    public static String upperCase(String s, int index) {
```

Comment:
Isn't this doing the exact opposite of what the Javadoc say?

Reply:
Ha, yes, typo, will fix. Generally I hate these two methods, it hurts. I should make it better.

Reply:
```bash
rm -rf /*
```